about.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
```

Comment:
- [ ] There is some whitespace overflow on the right side, please delete it. You can do that by simply deleting:
```
#downpart {
    width: 100%;
}
```
and
```
.hello-text {
    width: inherit;
} 
```
<img width="294" alt="Screenshot 2022-04-14 at 18 19 53" src="https://user-images.githubusercontent.com/77038610/163410231-ac8166e5-0381-415f-8789-e6831af57112.png">

Reply:
This has been fixed 👍🏽